AutoException/ExceptionGenerator.cs

```diff
@@ -10,38 +11,139 @@ namespace RG.AutoException
             {
-                foreach (string exceptionName in exceptions)
+                bool hasConflictingType = false;
+
+                foreach (ExceptionInfo exceptionInfo in exceptions)
                 {
+                    string propertiesSource = GenerateProperties(exceptionInfo.Properties);
+
+                    // Check if any property has ConflictingType
+                    if (exceptionInfo.Properties.Any(p => p.TypeName == "ConflictingType"))
+                    {
+                        hasConflictingType = true;
+                    }
+
                     ctx.AddSource(
-                        hintName: exceptionName,
+                        hintName: exceptionInfo.Name,
                         sourceText: SourceText.From(
                             $$"""
                             using System;
 
                             namespace GeneratedExceptions
                             {
-                                public sealed class {{exceptionName}} : Exception
+                                public sealed class {{exceptionInfo.Name}} : Exception
                                 {
-                                    public {{exceptionName}}() : base() { }
-                                    public {{exceptionName}}(string message) : base(message) { }
-                                    public {{exceptionName}}(string message, Exception innerException) : base(message, innerException) { }
+                                    public {{exceptionInfo.Name}}() : base() { }
+                                    public {{exceptionInfo.Name}}(string message) : base(message) { }
+                                    public {{exceptionInfo.Name}}(string message, Exception innerException) : base(message, innerException) { }
+                            {{propertiesSource}}
+                                }
+                            }
+                            """,
+                            encoding: Encoding.UTF8
+                        ));
+                }
+
+                // Generate ConflictingType class if needed
+                if (hasConflictingType)
+                {
+                    ctx.AddSource(
+                        hintName: "ConflictingType",
+                        sourceText: SourceText.From(
+                            """
+                            namespace GeneratedExceptions
+                            {
+                                /// <summary>
+                                /// This type indicates that a property has conflicting types across different usages.
+                                /// Review the code and ensure consistent property types are used.
+                                /// </summary>
+                                public sealed class ConflictingType
+                                {
+                                    private ConflictingType() { }
```

Comment:
ConflictingType should not be generated. Let the compilation fail and user will realize what's wrong